msbuild/Xamarin.Mac.Tasks.Core/Tasks/MmpTaskBase.cs

```diff
@@ -28,8 +28,6 @@ public class MmpTaskBase : BundlerToolTaskBase
 
 		public bool IsXPCService { get; set; }
 
-		public bool UseXamMacFullFramework { get; set; }
```

Comment:
was the `UseXamMacFullFramework` removal intentional ?
it's not mentioned in the commit message and this is specific to XM

Reply:
Yes, because it's not used anymore. I've updated the PR description and will add it to the commit message.